plugins/source/aws/resources/services/sqs/queues.go

```diff
@@ -62,169 +35,158 @@ func SQSQueues() *schema.Table {
 				Type:        schema.TypeString,
 				Resolver:    client.ResolveAWSRegion,
 			},
+			{
+				Name:     "tags",
+				Type:     schema.TypeJSON,
+				Resolver: resolveSqsQueueTags,
+			},
 			{
 				Name:        "url",
-				Description: "Queue URL",
+				Description: "The URL of the Amazon SQS queue.",
 				Type:        schema.TypeString,
 				Resolver:    schema.PathResolver("URL"),
 			},
-			{
-				Name:          "policy",
-				Description:   "The queue's policy. A valid Amazon Web Services policy.",
-				Type:          schema.TypeJSON,
-				IgnoreInTests: true,
-			},
-			{
-				Name:        "visibility_timeout",
-				Description: "The visibility timeout for the queue, in seconds.",
-				Type:        schema.TypeInt,
-			},
-			{
-				Name:        "maximum_message_size",
-				Description: "The limit of how many bytes a message can contain before Amazon SQS rejects it.",
-				Type:        schema.TypeInt,
-			},
-			{
-				Name:        "message_retention_period",
-				Description: "The length of time, in seconds, for which Amazon SQS retains a message.",
-				Type:        schema.TypeInt,
-			},
 			{
 				Name:        "approximate_number_of_messages",
 				Description: "The approximate number of messages available for retrieval from the queue.",
-				Type:        schema.TypeInt,
+				Type:        schema.TypeBigInt,
+			},
+			{
+				Name:        "approximate_number_of_messages_delayed",
+				Description: "The approximate number of messages in the queue that are delayed and not available for reading immediately.",
+				Type:        schema.TypeBigInt,
 			},
 			{
 				Name:        "approximate_number_of_messages_not_visible",
 				Description: "The approximate number of messages that are in flight.",
-				Type:        schema.TypeInt,
+				Type:        schema.TypeBigInt,
 			},
 			{
 				Name:        "created_timestamp",
-				Description: "UNIX time when the queue was created.",
-				Type:        schema.TypeTimestamp,
-				Resolver:    client.ResolveTimestampField("CreatedTimestamp"),
+				Description: "The time when the queue was created in seconds (epoch time).",
```

Comment:
Could we use a custom resolver to convert this (and other timestamps) to postgresql timestamp without timzeone? (UTC)?

It was requested a few times by customers to have a consistent format for timestamps..

Reply:
Ah, but actually I guess since it's just a small refactor then this is just optional (there are many places in the code with unix-timestamps instead of Postgres timestamps - out-of-scope of this PR)